src/components/nodes/logical/logical.js

```diff
@@ -0,0 +1,19 @@
+import PNConjunction from './And'
+import PNDisjunction from './Or'
+import PNExclusiveDisjunction from './Xor'
+import PNNegation from './Not'
+import PNImplication from './Imply'
+import PNBiconditional from './Xnor'
+import PNAlternativeDenial from './Nand'
+import PNJointDenial from './Nor'
```

Comment:
Why not rename these imports to their simpler versions too?

Reply:
I was wondering if maybe it is preferable to use the scientific names for the files, and only use simplified node titles...